src/styles/extra.css

```
@@ -38,3 +38,48 @@ html {
 select::-ms-expand {
   display: none;
 }
+
+/* Hover-revealed permalink anchors injected into Markdoc headings
+   (see src/components/HeadingAnchors.jsx). */
+.heading-anchor {
+  display: inline-flex;
+  align-items: center;
+  margin-left: 0.4em;
+  vertical-align: middle;
+  color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)